src/plugins/kibana_react/public/code_editor/languages/handlebars/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { LangModuleType } from '@kbn/monaco';
```

Comment:
I'm not getting much information from the word "Module" in this name. I'm used to thinking of modules in the `node_modules` sense, but not able to translate that meaning to this context. Is there a context that gives it meaning here?

This type seems to be a configuration object, specific to registering languages with our monaco package. I think a better name would be one that hints at that meaning, maybe `MonacoLanguageConfig`?

Reply:
As you can see, all languages in `@kbn/monaco` and in `plugins/kibana-react/public/code-editor` are represented as libraries, which encapsulate the complexity of configuring the language for `monaco`. They provide only the common interface (contract, if you want) to use. They are represented as libraries. That is a definition of the `module`, as I  think (not `node_modules`, but the definition of usual `module` in javascript).

What about `MonacoLanguageConfig`, I would prefer such a name, if there would be the only configuration for languages, but it also provides functions to getting errors and autocompletion in some languages. And it would confuse users (programmers), because in the terminology of `monaco` - `config` is just a part of configuring the language.